### PR TITLE
RA-1273: Remove config.require('patient') that's causing registration app to fail

### PR DESCRIPTION
**Pull request by andrewmoko**

### CODE REVIEW

**Comment by dkayiwa** [comment]
@andrewmoko did you see the merge conflicts here?

**Comment by andrewmoko** [comment]
All the changes from my PR  have been reverted. If I merge with master I stand to loose all that I have been working on. What's the correct process to go about this? Or should I start from scratch? Thanks

**Comment by dkayiwa** [comment]
Unfortunately it may be easier to just do a fresh one and put in your changes one by one.

**Comment by mogoodrich** [comment]
I'm not 100% sure, but wouldn't the easiest be to create a new branch on your fork, and apply a "revert" of the each of the four "reverts"? This should quickly get you back to the state you were in. Then make the additional changes and create a new pull request?

If you have to start entirely from scratch something is wrong in our process. :)

Mark Goodrich

________________________________

**Comment by andrewmoko** [comment]
Makes sense. I will give it a shot

**Comment by andrewmoko** [comment]
@dkayiwa here is the new PR. I will be closing this one. @mogoodrich Kindly review and possibly test too to make sure it works with your logic.

https://github.com/openmrs/openmrs-module-coreapps/pull/49

Thanks

File: omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java [hunk 1]
```diff
@@ -30,10 +31,26 @@ public void controller(FragmentConfiguration config,
 			@SpringBean("adtService") AdtService adtService,
 			@SpringBean("visitService") VisitService visitService,
 			@SpringBean("visitTypeHelper") VisitTypeHelper visitTypeHelper,
-			UiSessionContext sessionContext){
+			UiSessionContext sessionContext,
+			HttpServletRequest request,
+			@SpringBean("patientService")PatientService patientService){
 
-		config.require("patient");
 		Object patient = config.get("patient");
```
Comment (mogoodrich) [inline omod/src/main/java/org/openmrs/module/coreapps/fragment/controller/patientdashboard/VisitIncludesFragmentController.java]: I'm not 100% sure, but I think the problem is that the patient *is* required here, and was previously getting passed through via some other means.

See my comment on RA-1273 regarding the patient merge push--can you test if this change fixes that?

**Pull request closed without merging**